src/frameworks/index.ts

```diff
@@ -576,11 +589,12 @@ ${firebaseDefaults ? `__FIREBASE_DEFAULTS__=${JSON.stringify(firebaseDefaults)}\
       if (bootstrapScript) await writeFile(join(functionsDist, "bootstrap.js"), bootstrapScript);
 
       // TODO move to templates
+      const firstArg = frameworksBackend ? `${JSON.stringify(frameworksBackend)}, ` : "";
```

Comment:
Simplify to `JSON.stringify(frameworksBackend || {})`, that way space punc can stay in the onReq string template